poc/vdaf_poplar1.py

```diff
@@ -307,10 +306,10 @@ def unshard(Poplar1, agg_param,
 
     @classmethod
     def encode_agg_param(Poplar1, level, prefixes):
-        if level > 2 ** 16 - 1:
-            raise ERR_INPUT  # level too deep
-        if len(prefixes) > 2 ** 32 - 1:
-            raise ERR_INPUT  # too many prefixes
+        if level not in range(2**16 - 1):
+            raise ValueError('level out of range')
+        if len(prefixes) not in range(2**32 - 1):
+            raise ValueError('number of prefixes out of range')
```

Comment:
Same as in the document:
```suggestion
        if level not in range(2**16):
            raise ValueError('level out of range')
        if len(prefixes) not in range(2**32):
            raise ValueError('number of prefixes out of range')
```